### PR TITLE
[AE-4] Remove Unisense, and correct ESLOV caps in readme

### PR DESCRIPTION
**Pull request by aliphys**
### Description

Remove mention of Unisense and unify capitalisation of ESLOV in `readme.md`

### Pull request type

    [x] Fix
    [ ] Refactor
    [ ] Target update
    [ ] Functionality change
    [ ] Breaking change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,26 +9,29 @@ Nicla Sense ME belongs to Arduino Mbed OS boards and its core is available [here
...
 
 Table of Contents:
-- [Repo structure](#repo-structure)
-- [Main use cases](#main-use-cases)
-  * [Nicla standalone mode](#nicla-standalone-mode)
-  * [Control Nicla from an arduino board through eslov](#control-nicla-from-an-arduino-board-through-eslov)
-  * [Control Nicla from a PC - web server](#control-nicla-from-a-pc---web-server)
-  * [Control Nicla from a PC - command line tool](#control-nicla-from-a-pc---command-line-tool)
-  * [Update Nicla firmware](#update-nicla-firmware)
+- [Nicla](#nicla)
```
Comment (per1234) [inline README.md:12]: I don't think it makes sense to add the document title as a section in the ToC

**Review by per1234 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aliphys) [reply README.md:12]: This was here from before. I haven't added this as part of the PR.

Comment (aliphys) [inline README.md:12]:
```suggestion
```

**Comment by aliphys** [comment]
@per1234 Can you merge into `main`? I do not have write access.

**Pull request merged**